docs/app/data/components-page.json

```diff
@@ -9,8 +9,7 @@
                     "title": "Toggle Buttons",
                     "component": "ComponentsToggleButtonsComponent",
                     "version": "Angular",
-                    "externalUrl":
-                        "http://valor-software.com/ngx-bootstrap/#/buttons",
+                    "externalUrl": "http://valor-software.com/ngx-bootstrap/#/buttons",
```

Comment:
Hey, I understand, this could make merges much more complicated. I think i'll need to compare my settings with someone else to stop this happening as it auto happens on save for all files. Maybe there is a formatter I should be using for JSON files or a difference in config?

Reply:
For myself, typically I avoid doing a whole file reformat unless it's a real mess. I don't think your setting are necessarily wrong, it's just that this file has been changed a lot from 1.8 to 2.0 so I'd like to leave the big reformat until the changes are merged back into this repo.